### PR TITLE
add bearer token support

### PR DESCRIPTION
**Pull request by guoygang**

### CODE REVIEW

**Review by zhoney — DISMISSED** [review]

**Comment by imbajin** [comment]
image

seems ci failed

File: src/main/java/com/baidu/hugegraph/rest/AbstractRestClient.java [hunk 1]
```diff
@@ -547,4 +584,14 @@ public ClientConfig build() {
             return this.config;
         }
     }
+
+    public static class BearerRequestFilter implements ClientRequestFilter {
+        @Override
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/rest/AbstractRestClient.java:589]: add a blank line after class define

Comment (guoygang) [reply src/main/java/com/baidu/hugegraph/rest/AbstractRestClient.java:590]: done

**Comment by javeme** [comment]
close since created a new pr: https://github.com/hugegraph/hugegraph-common/pull/81

**Pull request closed without merging**